packages/toolpad-app/src/utils/readableDuration.ts

```diff
@@ -0,0 +1,34 @@
+const getReadableDuration = (editedAt: Date) => {
+  const duration = new Date().getTime() - editedAt.getTime();
+  const delta = Math.floor(duration / 1000);
+
+  const minute = 60;
+  const hour = minute * 60;
+  const day = hour * 24;
+  const week = day * 7;
+
+  let readableDuration = '';
+
+  if (delta < 30) {
+    readableDuration = 'just now';
+  } else if (delta < minute) {
+    readableDuration = `${delta} seconds ago`;
+  } else if (delta < 2 * minute) {
+    readableDuration = 'a minute ago';
+  } else if (delta < hour) {
+    readableDuration = `${Math.floor(delta / minute)} minutes ago`;
+  } else if (Math.floor(delta / hour) === 1) {
+    readableDuration = '1 hour ago';
+  } else if (delta < day) {
+    readableDuration = `${Math.floor(delta / hour)}  hours ago`;
+  } else if (delta < day * 2) {
```

Comment:
For the specific use case of the apps overview, I think we can stop at "x hours ago". there will be too many duplicates otherwise.

Reply:
What would apps created a few days or weeks ago say in that case?

Reply:
Just the date?

Reply:
Something like this?
<img width="727" alt="Screenshot 2022-06-23 at 2 39 20 PM" src="https://user-images.githubusercontent.com/19550456/175300469-914aec5c-9164-4bd7-b03a-43bda03bffea.png">